app/actors/iiif_print/actors/iiif_print_upload_actor.rb

```diff
@@ -47,17 +53,24 @@ def handle_issue_upload(env)
         work = env.curation_concern
         # must persist work to serialize job using it
         work.save!(validate: false)
-        user = env.current_ability.current_user.user_key
-        env.attributes[:admin_set_id] ||= default_admin_set
-        queue_job(work, @pdf_paths, user, env.attributes[:admin_set_id])
+        user = env.current_ability.current_user
+        admin_set = env.attributes[:admin_set_id] ||= default_admin_set
+        queue_job(work, @pdf_paths, user, admin_set, @prior_pdfs_count)
       end
 
-      def queue_job(work, paths, user, admin_set_id)
-        IiifPrint::CreatePagesJob.perform_later(
+      # submit the job
+      # @param [GenericWork, etc] A valid type of hyrax work
+      # @param [Array<String] paths to PDF attachments
```

Comment:
getting nit picky i know, but should it be `[Array<String>]`? missing a `>`, not sure how yardoc would interpret it